tests/integration/targets/ldap_search/aliases

```diff
@@ -0,0 +1,6 @@
+skip/aix
```

Comment:
You need to specify a group, otherwise the tests are not run at all:
```suggestion
shippable/posix/group1
skip/aix
```

Reply:
Thanks, I was wondering why it didn't execute :-)